### DISCUSSION TITLE
Ramp down of website and app

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3303] [General] [answered]
Since the planning for ramping down the website and app has now started I will close off my issues here which I no longer expect to get resolved in the remaining time-frame.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Did I miss an official announcement reg. the ramp down period?

**Reply by Ein-Tim** [reply]
Ah, I see: https://github.com/corona-warn-app/cwa-website/pull/3302#issuecomment-1369019402

**Reply by MikeMcC399** [reply]
No dates yet, but I am assuming that it will happen in the next months. So for any of my issues which have been open for months and years, I will close these unless I think it is important that any are still addressed.

**Reply by MikeMcC399** [reply]
Is there any new planning date for ramp-down of the website?

**Comment by MikeMcC399** [comment, marked as answer]
The question is answered in https://github.com/corona-warn-app/cwa-app-android/pull/5885#issuecomment-1457988000 for the app. I assume that the website will follow the same dates.
